compiled.scss

```diff
@@ -68,6 +68,12 @@ $grid-breakpoint-attribute-prefix: "u-" !default;
     @include grid--order-reverse();
   }
 
+  // ------ Wrapping -------
+
+  .grid[#{$attr}~="wrap:nowrap"] {
```

Comment:
Is there a way to override the `nowrap`? e.g. at `xs` it might be `nowrap` but at `md` it might be `wrap`... Is that something we should handle?